src/DETHRACE/common/depth.c

```diff
@@ -137,7 +148,11 @@ br_scalar EdgeU(br_angle pSky, br_angle pView, br_angle pPerfect) {
     br_scalar b;
     br_scalar c;
     LOG_TRACE("(%d, %d, %d)", pSky, pView, pPerfect);
-    NOT_IMPLEMENTED();
+
+    a = cosf(BrAngleToRadian(pPerfect));
+    b = sinf(BrAngleToRadian(pView));
+    c = cosf(BrAngleToRadian(pView));
+    return b * a * a / (BrAngleToRadian(pSky) * (1.f + c));
```

Comment:
Please take a close look at this function as I'm not sure of it.

Reply:
looks good to me